src/containers/Home.tsx

```diff
@@ -59,28 +55,19 @@ const Home: FunctionComponent<Props> = ({ workspaceInfo, serverInfo, width, heig
           width={W}
           height={H - headerHeight}
           defaultFeedId={serverInfo.defaultFeedId || ''}
-          {...{workspaceInfo, sortings, recordings, onDeleteRecordings, onDeleteSortings, plugins, history, location}}
+          {...{workspaceInfo, workspace, workspaceDispatch, plugins, history, location}}
         />
       </div>
     </div>
   );
 }
 
 const mapStateToProps: MapStateToProps<StateProps, OwnProps, RootState> = (state: RootState, ownProps: OwnProps): StateProps => ({
-  sortings: state.sortings,
-  recordings: state.recordings,
   plugins: filterPlugins(state.plugins),
   serverInfo: state.serverInfo
 })
   
 const mapDispatchToProps: MapDispatchToProps<DispatchProps, OwnProps> = (dispatch: Dispatch<RootAction>, ownProps: OwnProps) => ({
-  onDeleteRecordings: (recordingIds: string[]) => {
```

Comment:
No longer using redux for managing recordings/sortings. Those are managed in the workspace/workspaceDispatch.